StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1120CodeFixProvider.cs

```diff
@@ -113,12 +96,12 @@ private static bool TriviaHasLeadingContentOnLine(SyntaxNode root, IReadOnlyList
             return false;
```

Comment:
The if and return statements above can be replaced by a single return statement.